"""Code generated by Speakeasy (https://speakeasyapi.dev). DO NOT EDIT."""

from __future__ import annotations
from enum import Enum

class FailbackReplicationErrorEnum(str, Enum):
    AGENT_NOT_SEEN = 'AGENT_NOT_SEEN'
    FAILBACK_CLIENT_NOT_SEEN = 'FAILBACK_CLIENT_NOT_SEEN'
    NOT_CONVERGING = 'NOT_CONVERGING'
    UNSTABLE_NETWORK = 'UNSTABLE_NETWORK'
    FAILED_TO_ESTABLISH_RECOVERY_INSTANCE_COMMUNICATION = 'FAILED_TO_ESTABLISH_RECOVERY_INSTANCE_COMMUNICATION'
    FAILED_TO_DOWNLOAD_REPLICATION_SOFTWARE_TO_FAILBACK_CLIENT = 'FAILED_TO_DOWNLOAD_REPLICATION_SOFTWARE_TO_FAILBACK_CLIENT'
    FAILED_TO_CONFIGURE_REPLICATION_SOFTWARE = 'FAILED_TO_CONFIGURE_REPLICATION_SOFTWARE'
    FAILED_TO_PAIR_AGENT_WITH_REPLICATION_SOFTWARE = 'FAILED_TO_PAIR_AGENT_WITH_REPLICATION_SOFTWARE'
    FAILED_TO_ESTABLISH_AGENT_REPLICATOR_SOFTWARE_COMMUNICATION = 'FAILED_TO_ESTABLISH_AGENT_REPLICATOR_SOFTWARE_COMMUNICATION'
    FAILED_GETTING_REPLICATION_STATE = 'FAILED_GETTING_REPLICATION_STATE'
    SNAPSHOTS_FAILURE = 'SNAPSHOTS_FAILURE'
    FAILED_TO_CREATE_SECURITY_GROUP = 'FAILED_TO_CREATE_SECURITY_GROUP'
    FAILED_TO_LAUNCH_REPLICATION_SERVER = 'FAILED_TO_LAUNCH_REPLICATION_SERVER'
    FAILED_TO_BOOT_REPLICATION_SERVER = 'FAILED_TO_BOOT_REPLICATION_SERVER'
    FAILED_TO_AUTHENTICATE_WITH_SERVICE = 'FAILED_TO_AUTHENTICATE_WITH_SERVICE'
    FAILED_TO_DOWNLOAD_REPLICATION_SOFTWARE = 'FAILED_TO_DOWNLOAD_REPLICATION_SOFTWARE'
    FAILED_TO_CREATE_STAGING_DISKS = 'FAILED_TO_CREATE_STAGING_DISKS'
    FAILED_TO_ATTACH_STAGING_DISKS = 'FAILED_TO_ATTACH_STAGING_DISKS'
    FAILED_TO_PAIR_REPLICATION_SERVER_WITH_AGENT = 'FAILED_TO_PAIR_REPLICATION_SERVER_WITH_AGENT'
    FAILED_TO_CONNECT_AGENT_TO_REPLICATION_SERVER = 'FAILED_TO_CONNECT_AGENT_TO_REPLICATION_SERVER'
    FAILED_TO_START_DATA_TRANSFER = 'FAILED_TO_START_DATA_TRANSFER'
